restler/restler_settings.py

```diff
@@ -704,12 +706,54 @@ def allow_grammar_py_user_update(self):
 
     @property
     def token_refresh_cmd(self):
-        return self._token_refresh_cmd.val
+        if self._token_refresh_cmd.val:
+            return self._token_refresh_cmd.val
+        elif 'token' in self._authentication_settings.val and 'cmd' in self._authentication_settings.val['token']:
+            return self._authentication_settings.val['token']['cmd']
+        else:
+            return None
 
     @property
     def token_refresh_interval(self):
-        return self._token_refresh_interval.val
+        if self._token_refresh_interval.val:
+            return self._token_refresh_interval.val
+        elif 'token' in self._authentication_settings.val and 'token_refresh_interval' in self._authentication_settings.val['token']:
```

Comment:
>elif 'token' in self._authentication_settings.val and  [](http://example.com/codeflow?start=8&length=54)

Could you please break these up across multiple lines?

Reply:
changing these